meta/main.yml

```diff
@@ -0,0 +1,17 @@
+galaxy_info:
+  author: Andrew Euredjian
+  description: Role to install cockpit to a set of hosts and start the service
+  company: Red Hat, Inc.
+  license: GPLv3
+  min_ansible_version: 2.4
+  platforms:
+    - name: EL
+      versions:
+        - 7
+
+  galaxy_tags:
+    - oasis
+
+dependencies:
```

Comment:
Are we sure we want to set these as dependencies? I'm not sure this role truly depends on them. For one, it doesn't call them in the test converge.yml file, which means either this dependency is more of a suggestion or that converge step in Molecule won't work properly.

Reply:
@seandst - Would that mean essentially taking the playbook from the downstream role and using that as the example for the upstream README?

Reply:
I think so, since the existing downstream scenario is an "ideal" install, and the most likely type of install (I think, at least) that users of this role are likely to do. Sanitize the dirty redhat secret launch codes, of course. :)

I do recommend breaking it up into two sections, with one focused on setting up the required repositories with RHSM and another focused on exposting the service at either its normal port (9090?) or a different custom port.

Reply:
What information is considered the secret launch codes? I figure username and password would be but are there any others? (i.e. server_hostname, pool_ids, repositories, etc.)